admin/app/views/pressInteractive.scala.html

```diff
@@ -45,6 +45,15 @@ <h4 id=progress-status>Progress of pressing</h4>
         <br>
         <div id="server-message">No pressing in progress</div>
     </div>
+  <br>
+  <br>
+    <h2>Un-press an interactive</h2>
+    <br>
+    <ul>
+        <li>
+            Un-pressing is not currently an automated process. Please contact the dotcom team if you need to un-press an individual interactive.
```

Comment:
`dotcom team (dotcom.platform@theguardian.com)` <- might be worth adding the team email just in case people don't know it.

Reply:
Great point, will add!